Manito/Manito/Screens/Interaction/OpenManittoViewController.swift

```diff
@@ -53,59 +47,67 @@ final class OpenManittoViewController: BaseViewController {
     }()
     private let titleLabel: UILabel = {
         let label = UILabel()
-        label.text = TextLiteral.openManittoViewController
         label.font = .font(.regular, ofSize: 34)
+        label.text = TextLiteral.openManittoViewControllerTitle
         return label
     }()
-    private var scrollNumberIndex = -1 {
+
+    // MARK: - property
+
+    private let openManittoService: DetailIngAPI = DetailIngAPI(apiService: APIService())
+
+    private var manittoRandomIndex = -1 {
```

Comment:
p2;

```suggestion
    private var manittoRandomIndex: Int = -1 {
```
타입 명시가 빠져있군요..

Reply:
[0c8e235](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/427/commits/0c8e235c5931cb721ae96aa1c349843beb873655)

타입 명시했습니다.